### DISCUSSION TITLE
[Question] Which one is better to use autoortho, autoortho_win.exe or autoortho_win.zip?

### DISCUSSION THREAD

**Discussion by AntonKuf** [kubilus1/autoortho#261] [Q&A]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: [e.g. Windows 11]
- XPlane Version: [e.g. XP12 12.05]
- AutoOrtho Version [e.g. 0.4.4]
- Filesystem type (NTFS)

**Additional context**
Add any other context about the problem here. (scenery version if applicable, etc, logfile snippet, etc)

### question

Which one is better to use autoortho, autoortho_win.exe or autoortho_win.zip? what are the differences?

**Comment by dpdurst** [comment]
I'll just say, AutoOrtho_win.exe is everything you need just run the program from the folder you drop it in and your set. Simple.
